doc/reference/reference_lua/uri.rst

```diff
@@ -27,7 +28,7 @@ has three components:
 * ``tarantool.org/x.html`` is the path,
```

Comment:
> For example the string ``'https://tarantool.org/x.html#y'``
> has three components:

* ``https`` is the scheme,
* ``tarantool.org/x.html`` is the path,
* ``y`` is the fragment.

Suggestion: 
For example, the string ``'https://tarantool.org/x.html#y'`` has three components:

* ``https`` is the scheme.
* ``tarantool.org/x.html`` is the path.
* ``y`` is the fragment.